go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
 	github.com/XSAM/otelsql v0.33.0
 	github.com/alicebob/miniredis/v2 v2.33.0
+	github.com/dgraph-io/dgo/v210 v210.0.0-20230328113526-b66f8ae53a2d
```

Comment:
This is not required as we are adding dgraph as an external datasource

Reply:
Yups removed